scripts-available/CDB_Groups.sql

```diff
@@ -0,0 +1,138 @@
+-- Creates a new group
+CREATE OR REPLACE
+FUNCTION cartodb.CDB_Group_CreateGroup(group_name text)
+    RETURNS VOID AS $$
+DECLARE
+    cdb_group_role TEXT;
+BEGIN
+    cdb_group_role := cartodb._CDB_Group_GroupRole(group_name);
+    IF NOT EXISTS ( SELECT 1 FROM pg_roles WHERE rolname = cdb_group_role )
+    THEN
+        EXECUTE 'CREATE ROLE "' || cdb_group_role || '" NOLOGIN;';
```

Comment:
Hey, organization role is not lazily created it's created when the extension is installed the first time: https://github.com/CartoDB/cartodb-postgresql/blob/0.8.2/scripts-available/CDB_Organizations.sql#L7-L17

I would do the same for admin role.